app/views/payments/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<% content_for :title, "Betalingen - SOFIA" %>
+
+<div class="container footer-padding">
+  <div class="row pt-4">
+    <div class="col-sm-12 d-flex justify-content-between align-items-center">
+      <h1>Betalingen</h1>
+    </div>
+
+    <table class="table table-striped mt-2">
+      <thead>
+      <tr>
+        <th scope="col">#</th>
+        <th scope="col">Status</th>
+        <th scope="col">Bedrag</th>
+        <th scope="col">Gebruiker</th>
+      </tr>
+      </thead>
+      <tbody>
+      <% @payments.each do |payment| %>
+        <tr>
+          <th scope="row"><%= payment[:id] %></th>
+          <td><%= payment[:status] %></td>
```

Comment:
Maybe we want to show the status to the user in a more user-friendly way instead of just showing the enum value (i.e. 'Betaald' instead of 'paid')? If we really want to be fancy we can also indicate the status of the order by text color (i.e. paid is green, pending is orange, failed is red)?